.ci/pitest.sh

```diff
@@ -135,8 +135,8 @@ pitest-ant)
 pitest-indentation)
   mvn -e -P$1 clean test org.pitest:pitest-maven:mutationCoverage;
   declare -a ignoredItems=(
+  "AbstractExpressionHandler.java.html:<td class='covered'><pre><span  class='survived'>                &#38;&#38; tree.getColumnNo() &#60; realStart.getColumnNo()) {</span></pre></td></tr>"
```

Comment:
we do not allow new problems with coverage to appear.
This list of ignored problems can only be reduced.

Reply:
This is having conditional mutations survived, 
For removing this mutation, we will need test case that has  two different AST expressions with  same line Number and Same Column Number.
Can two different AST have same line and column Number?...I was unable to find that case.

Reply:
ok, agree, it is possible, but I do not remember exact case.